src/VscPreviewExtension/package.json

```diff
@@ -3,22 +3,35 @@
   "displayName": "DfmPreview",
   "description": "Preview the DocFX Flavored Markdown and token tree in Visual Studio Code",
   "version": "0.0.21",
-  "publisher": "928pjy",
+  "publisher": "docfxsvc",
   "engines": {
     "vscode": "^1.3.0"
   },
   "categories": [
     "Languages",
     "Other"
   ],
+  "main": "./out/src/extension",
+  "icon": "img/logo.png",
+  "galleryBanner": {
+    "color": "#2D2D2D",
+    "theme": "dark"
+  },
+  "bugs": {
+    "url": "https://github.com/dotnet/docfx/issues"
+  },
+  "homepage": "https://github.com/dotnet/docfx/blob/dev/src/VscPreviewExtension/README.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dotnet/docfx.git"
```

Comment:
>https://github.com/dotnet/docfx.git [](start = 12, length = 35)

will it be confused if user entering docfx repo instead of dfm preview extension folder? Is it possible to use git hub sub folder instead?